CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 # Opera Changelog
 
-## 0.1.0 - September 12, 2020
+### 0.2.3 - July 26, 2021
 
-- Initial release
+- Support context, params and dependencies accessors
+- Removed depreceted `finish` method
```

Comment:
if `finish` is removed, please explaining the changeling briefly why and what should we use instead of it. I honestly don't know that and it was never communicated. Users of the gem would feel the same.

Reply:
no, we had `Warning` before:
https://github.com/Profinda/opera/pull/5/files#diff-51d3653f611ad330e64b154eab9ab5d59fa90bfe203af867b9a754a1f7c96c97L28

You probably didn't see it because we already were using `finish!`

Reply:
https://github.com/Profinda/opera/pull/5/files#diff-51d3653f611ad330e64b154eab9ab5d59fa90bfe203af867b9a754a1f7c96c97L28